build.gradle

```diff
@@ -409,7 +410,7 @@ task collectDocumentation(group: "documentation", description: "Copy the documen
 }
 
 gitPublish {
-
+    
```

Comment:
Adding some trailing whitespace here

Reply:
it was a copy/paste/modify from the developer example. Do you just mean it's an extra new line or that there are space characters somewhere?

Reply:
Yes it's a line with 4 spaces